reco_utils/README.md

```diff
@@ -0,0 +1,37 @@
+# Recommender Utilities
+
+This module (reco_utils) contains functions to simplify common tasks used when developing and evaluating recommender systems. A short description of the sub-modules is provided below. For more details about what functions are available and how to use them, please review the doc-strings provided with the code.
+
+## Sub-Modules
+
+### Common
+This submodule contains high-level utilities for defining constants used in most algorithms as well as helper functions for managing aspects of different frameworks: gpu, spark, jupyter notebook.
+
+### Dataset
+Dataset includes helper functions for interacting with Azure Cosmos databases, pulling different sizes of the Movielens dataset and formatting them appropriately as well as utilities for splitting data for training / testing.
+
```

Comment:
Do we want to have a level-4 title here, something like "data loading" or "data importing"?

Reply:
sounds good